smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSource.java

```diff
@@ -140,11 +144,11 @@ public KafkaSource(Vertx vertx,
                                 .with(
                                         a -> {
                                             log.executedConsumerAssignedRebalanceListener(group);
-                                            kafkaConsumer.resume();
+                                            kafkaConsumer.fetch(resumeFetchAmount);
```

Comment:
What happens when the amount is consumed? Would it get more?

Reply:
It will get more. That's the current behavior without the consumer re-balance.